src/components/KTableView/KTableView.cy.ts

```diff
@@ -481,6 +481,22 @@ describe('KTableView', () => {
         })
       })
     })
+
+    it('should respect initial sort order from table preferences', () => {
```

Comment:
Can you assert the table is actually sorted correctly here as well by the column?

Would also be nice to have negative path tests, e.g. table shows correct icon when no initial sort is applied, etc.